packages/workspace/src/utils/runtime-lint-utils.ts

```diff
@@ -205,3 +205,18 @@ export function onlyLoadChildren(
 export function getSourceFilePath(sourceFileName: string, projectPath: string) {
   return normalize(sourceFileName).substring(projectPath.length + 1);
 }
+
+/**
+ * Verifies whether the given node has an architect builder attached
+ * @param projectGraph the node to verify
+ */
+export function hasArchitectBuildBuilder(
+  projectGraph: ProjectGraphNode
+): boolean {
+  return (
+    // can the architect not be defined? real use case?
+    projectGraph.data.architect &&
```

Comment:
@vsavkin I was especially wondering about the `data.architect` property. It is not in the `ProjectGraphNode` type, so I assume it could be not there? Is that even a real use case?